ansible/roles/supervisord/files/supervisord.conf

```diff
@@ -0,0 +1,139 @@
+; Sample supervisor config file.
```

Comment:
Anything obviously wrong here? I want supervisorctl to only be accessible from the server. If we enable the web mode we will need to use vault to hide the user / password.

Reply:
The docs say if the `inet_http_server` section is omitted, then supervisor will not start the HTTP server so yeah you're fine.